libs/@guardian/libs/src/index.ts

```diff
@@ -31,6 +31,8 @@ export { isUndefined } from './isUndefined/isUndefined';
 
 export { joinUrl } from './joinUrl/joinUrl';
 
+export { literalUnionPredicate } from './literalUnionPredicate/literalUnionPredicate';
```

Comment:
i'm not sure how ergonomic/accessible this name will be to people newly learning TS (i had to read the example to see what it did).

given we have things like `isString`, could we use less technical language? e.g. `isOneOf`?

Reply:
I originally went with simply `is`! Naming things is hard and suggestions are welcome and hope to get the best thing before merging. Previous thoughts before I settled on the currently overly explicit `literalUnionPredicate`:

- `is`
- `isLiteral`
- `isOneOf`
- `isIn`
- `guard`

---

Wondering if this requires a change request, though? There’s not been an approval from CSTI yet. Changing names can also be done in a non-breaking ways, as was done for [Pillar](https://github.com/guardian/csnx/pull/680), so the cost of renaming is not unsurmoutable.

Reply:
I'd vote for `isOneOf`, it seems to me to best describe what you've learned about the candidate?

would be good to get a name that seems good before merging. no problem with making changes if it turns out be a bad one, but we should do that because we discovered something, not because we could decide wink 😜

Reply:
`isOneOf` it is!